src/docs/binding-structs.md

```diff
@@ -0,0 +1,100 @@
+# Binding Structs
+
+Structs in Swift are value types that can have a number of forms each of which requires a certain amount of consideration: frozen/non-frozen and blitable and non-blitable.
```

Comment:
```suggestion
Structs in Swift are value types that can have a number of forms each of which requires a certain amount of consideration: frozen/non-frozen and blittable and non-blittable.
```

Reply:
You can add a reference to the .NET documentation - https://learn.microsoft.com/dotnet/framework/interop/blittable-and-non-blittable-types.